src/main/java/com/o19s/es/ltr/feature/store/StoredLtrModel.java

```diff
@@ -164,7 +165,9 @@ public XContentBuilder toXContent(XContentBuilder builder, Params params) throws
         if (modelAsString) {
             builder.value(rankingModel);
         } else {
-            try (XContentParser parser = JsonXContent.jsonXContent.createParser(EMPTY, rankingModel)) {
+            try (XContentParser parser = JsonXContent.jsonXContent.createParser(EMPTY,
+                    LoggingDeprecationHandler.INSTANCE, rankingModel)
```

Comment:
again, sorry for the might be stupid question. Why this is logged as deprecated?

Reply:
This is the new signature for createParser, I suppose that now they force us to specify the deprecation policy. (this is part of https://github.com/elastic/elasticsearch/issues/28504 & https://github.com/elastic/elasticsearch/pull/28697)